[31-Dec-2019 18:17:32 UTC] PHP Warning:  session_start(): Cannot start session when headers already sent in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php on line 3
[13-Aug-2020 05:32:15 UTC] PHP Warning:  A non-numeric value encountered in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php on line 30
[13-Aug-2020 05:32:58 UTC] PHP Warning:  A non-numeric value encountered in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php on line 30
[22-Aug-2020 09:41:36 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(78): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 09:42:04 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(78): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 09:42:24 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(78): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 09:44:10 UTC] PHP Notice:  Undefined index: q in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php on line 41
[22-Aug-2020 09:44:11 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "VALUE_EMPTY","detail": "Field must not be blank","field": "source_id","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"idempotency_k...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(78): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 09:44:21 UTC] PHP Notice:  Undefined index: q in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php on line 41
[22-Aug-2020 09:44:22 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "VALUE_EMPTY","detail": "Field must not be blank","field": "source_id","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"idempotency_k...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(78): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 09:45:48 UTC] PHP Notice:  Undefined index: q in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php on line 44
[22-Aug-2020 09:45:49 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "VALUE_EMPTY","detail": "Field must not be blank","field": "source_id","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"idempotency_k...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 09:46:02 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 09:46:11 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 09:46:43 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(78): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 09:46:54 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 11:02:37 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 11:28:26 UTC] PHP Notice:  Undefined index: order_info_row in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethodForm.php on line 14
[22-Aug-2020 11:28:26 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethodForm.php on line 14
[22-Aug-2020 12:33:20 UTC] PHP Notice:  Undefined index: order_info_row in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethodForm.php on line 14
[22-Aug-2020 12:33:20 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethodForm.php on line 14
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: user_info_row in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 29
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 56
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: doc_type in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 57
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: academic_lvl in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 58
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: no_of_pages in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 59
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: deadline_time in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 60
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: deadline_date in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 60
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 61
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: title in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 62
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: subject in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 63
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: citation_style in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 64
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: no_of_sources in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 65
[22-Aug-2020 12:33:31 UTC] PHP Notice:  Undefined index: description in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 66
[22-Aug-2020 12:35:08 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[22-Aug-2020 12:52:49 UTC] PHP Notice:  Undefined index: order_info_row in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethodForm.php on line 14
[22-Aug-2020 12:52:49 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethodForm.php on line 14
[25-Aug-2020 05:12:46 Asia/Karachi] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 56
[25-Aug-2020 05:12:46 Asia/Karachi] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 61
[25-Aug-2020 00:14:45 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[25-Aug-2020 00:16:50 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[25-Aug-2020 00:18:07 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[28-Aug-2020 22:03:01 UTC] PHP Notice:  Undefined index: order_info_row in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethodForm.php on line 52
[28-Aug-2020 22:03:01 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethodForm.php on line 52
[03-Oct-2020 14:32:11 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[27-Oct-2020 22:05:54 UTC] PHP Notice:  Undefined index: order_info_row in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethodForm.php on line 52
[27-Oct-2020 22:05:54 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethodForm.php on line 52
[28-Oct-2020 19:31:18 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_header() in /home/perfecta/public_html/Perfectapapers_Custom_Files/order-now.php:16
Stack trace:
#0 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/order-now.php on line 16
[28-Oct-2020 19:31:22 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function get_header() in /home/perfecta/public_html/Perfectapapers_Custom_Files/order-now.php:16
Stack trace:
#0 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/order-now.php on line 16
[04-Nov-2020 01:32:29 Asia/Karachi] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 56
[04-Nov-2020 01:32:29 Asia/Karachi] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 61
[03-Nov-2020 20:32:59 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Undefined index: user_info_row in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 29
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Undefined index: order_info_row in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 30
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 56
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 57
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 58
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 59
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 60
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 60
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 61
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 62
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 63
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 64
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 65
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 66
[10-Nov-2020 18:43:17 UTC] PHP Notice:  Trying to access array offset on value of type null in /home/perfecta/public_html/Perfectapapers_Custom_Files/review_order_pg.php on line 67
[23-Feb-2021 09:18:13 UTC] PHP Warning:  A non-numeric value encountered in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php on line 36
[23-Feb-2021 09:18:14 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "VALUE_EMPTY","detail": "Field must not be blank","field": "source_id","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":""...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[23-Feb-2021 09:30:38 UTC] PHP Warning:  A non-numeric value encountered in /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php on line 36
[23-Feb-2021 09:30:38 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "VALUE_EMPTY","detail": "Field must not be blank","field": "source_id","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":""...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
[01-Mar-2021 21:04:12 UTC] PHP Fatal error:  Uncaught SquareConnect\ApiException: [HTTP/2 400] {"errors": [{"code": "BAD_REQUEST","detail": "This account has not been enabled to take payments","category": "INVALID_REQUEST_ERROR"}]}
 in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php:236
Stack trace:
#0 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(427): SquareConnect\ApiClient->callApi('/v2/payments', 'POST', Array, '{"source_id":"c...', Array, '\\SquareConnect\\...')
#1 /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/Api/PaymentsApi.php(363): SquareConnect\Api\PaymentsApi->createPaymentWithHttpInfo(Object(SquareConnect\Model\CreatePaymentRequest))
#2 /home/perfecta/public_html/Perfectapapers_Custom_Files/PaymentMethod.php(81): SquareConnect\Api\PaymentsApi->createPayment(Object(SquareConnect\Model\CreatePaymentRequest))
#3 {main}
  thrown in /home/perfecta/public_html/Perfectapapers_Custom_Files/vendor/square/connect/lib/ApiClient.php on line 236
